mmv1/products/dataplex/Task.yaml

```diff
@@ -0,0 +1,487 @@
+            output: true
+            description: |
+              The number of times the job has been retried (excluding the initial attempt).
+          - !ruby/object:Api::Type::Enum
+            name: 'service'
+            output: true
+            description: |
+              The underlying service running a job.
+            values:
+              - :SERVICE_UNSPECIFIED
+              - :DATAPROC
+          - !ruby/object:Api::Type::String
+            name: 'serviceJob'
+            output: true
+            description: |
+              The full resource name for the job run under a particular service.
+          - !ruby/object:Api::Type::String
+            name: 'message'
+            output: true
+            description: |
+              Additional information about the current state.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'spark'
+    description: |
+      A service with manual scaling runs continuously, allowing you to perform complex initialization and rely on the state of its memory over time.
+    exactly_one_of:
+      - spark
+      - notebook
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: 'fileUris'
+        description: |
+          Cloud Storage URIs of files to be placed in the working directory of each executor.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Array
+        name: 'archiveUris'
+        description: |
+          Cloud Storage URIs of archives to be extracted into the working directory of each executor. Supported file types: .jar, .tar, .tar.gz, .tgz, and .zip.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
+        name: 'infrastructureSpec'
+        description: |
+          Infrastructure specification for the execution.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'batch'
+            description: |
+              Compute resources needed for a Task when using Dataproc Serverless.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'executorsCount'
+                default_value: 2
+                description: |
+                  Total number of job executors. Executor Count should be between 2 and 100. [Default=2]
+              - !ruby/object:Api::Type::Integer
+                name: 'maxExecutorsCount'
+                default_value: 1000
+                description: |
+                  Max configurable executors. If maxExecutorsCount > executorsCount, then auto-scaling is enabled. Max Executor Count should be between 2 and 1000. [Default=1000]
+          - !ruby/object:Api::Type::NestedObject
+            name: 'containerImage'
+            description: |
+              Container Image Runtime Configuration.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'image'
+                description: |
+                  Container image to use.
```

Comment:
Is there a specific set of allowed options for this field?

Reply:
No user can provide his own value

Reply:
Is there a required format? Like referencing a URI or other cloud resource url?

Reply:
Yes there is a referencing URI but don't think there is any other data-type better suited for the parameter.

Reply:
Can you update the field description to say that is referring to a URI? Thanks!